airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresTestDatabase.java

```diff
@@ -57,6 +57,7 @@ private ContainerModifier(String methodName) {
 
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
??

Reply:
I'm not quite sure, but this underlying library seems to have a deprecation that is failing the gradle check, so it i's necessary